### PR TITLE
Remove extensions

### PR DESCRIPTION
**Pull request by akshay111meher**

### CODE REVIEW

File: contracts/Pool/Repayments.sol [hunk 1]
```diff
@@ -22,7 +22,7 @@ contract Repayments is Initializable, IRepayment, ReentrancyGuardUpgradeable {
 
     uint256 constant MAX_INT = 2**256 - 1;
     uint256 constant YEAR_IN_SECONDS = 365 days;
-    uint256 constant SCALING_FACTOR = 1e18;
+    uint256 public constant SCALING_FACTOR = 1e18;
```
Comment (prateekreddy) [inline contracts/Pool/Repayments.sol:25]: Public not necessary here

File: contracts/Pool/Repayments.sol [hunk 2]
```diff
@@ -316,12 +307,11 @@ contract Repayments is Initializable, IRepayment, ReentrancyGuardUpgradeable {
         return _interestLeft;
     }
 
-    /// @notice Given there is no loan extension, find the overdue interest after missing the repayment deadline
+    /// @notice find the overdue interest after missing the repayment deadline
     /// @dev (SCALING_FACTOR) is included to maintain the accuracy of the arithmetic operations
     /// @param _poolID address of the pool
     /// @return interest amount that is overdue
     function getInterestOverdue(address _poolID) public view returns (uint256) {
```
Comment (prateekreddy) [inline contracts/Pool/Repayments.sol:314]: This function is not needed

File: contracts/PooledCreditLine/LenderPool.sol [hunk 3]
```diff
@@ -234,64 +224,71 @@ contract LenderPool is ERC1155Upgradeable, ReentrancyGuard {
...
 
         creditLines[_id].lenders[_lender].yieldInterestWithdrawnShares += _yieldInterestForLender;
         creditLines[_id].lenders[_lender].borrowerInterestSharesWithdrawn += _borrowerInterestForLender;
         creditLines[_id].borrowerInterestSharesWithdrawn += _borrowerInterestForLender;
         creditLines[_id].yieldInterestWithdrawnShares += _yieldInterestForLender;
         creditLines[_id].sharesHeld -= (_yieldInterestForLender + _borrowerInterestForLender);
 
-        uint256 _interestToWithdraw = IYield(_strategy).getTokensForShares(_yieldInterestForLender + _borrowerInterestForLender, _lentToken);
+        uint256 _interestToWithdraw = IYield(_strategy).getTokensForShares(
+            _yieldInterestForLender + _borrowerInterestForLender,
```
Comment (prateekreddy) [inline contracts/PooledCreditLine/LenderPool.sol:269]: use .add just to be sure

File: contracts/Verification/twitterVerifier.sol [hunk 4]
```diff
@@ -116,6 +113,31 @@ contract TwitterVerifier is Initializable, IVerifier, OwnableUpgradeable, EIP712
         emit UserRegistered(msg.sender, _isMasterLinked, _twitterId);
     }
 
+    function registerUserViaOwner(
```
Comment (prateekreddy) [inline contracts/Verification/twitterVerifier.sol:116]: Remove this function

File: contracts/interfaces/IRepayment.sol [hunk 5]
```diff
@@ -54,9 +54,7 @@ interface IRepayment {
 
     function getCurrentLoanInterval(address poolID) external view returns (uint256 scaledCurrentInterval);
 
-    function instalmentDeadlineExtended(address _poolID) external;
-
-    function didBorrowerDefault(address _poolID) external view returns (bool isBorrowerDefaulter);
+    function didBorrowerDefault(address _poolID) external view returns (bool);
```
Comment (prateekreddy) [inline contracts/interfaces/IRepayment.sol:57]: name for return variable should be present

File: contracts/yield/CompoundYield.sol [hunk 6]
```diff
@@ -21,7 +21,7 @@ contract CompoundYield is IYield, Initializable, OwnableUpgradeable, ReentrancyG
     using SafeERC20 for IERC20;
     using SafeMath for uint256;
 
-    address public constant TREASURY = address(0);
+    address public constant TREASURY = address(0xee);
```
Comment (prateekreddy) [inline contracts/yield/CompoundYield.sol:24]: Add a TODO that it is going to be updated before deployment

**Review by prateekreddy — CHANGES_REQUESTED** [review]
Please take a look at above comments

File: contracts/Pool/Repayments.sol (hunk 1 above)
Comment (akshay111meher) [reply contracts/Pool/Repayments.sol:25]: help's a getters in testing. 
if any gas issues, we can remove it.

**Pull request merged**